src/app/modules/main/wallet_section/all_collectibles/controller.nim

```diff
@@ -47,7 +47,7 @@ proc init*(self: Controller) =
     self.delegate.refreshNetworks()
 
 proc getWalletAddresses*(self: Controller): seq[string] =
-  return self.walletAccountService.getWalletAddresses()
```

Comment:
The function name makes me think this should work :D
It probably need to be renamed to know what behavior to expect.